INFO     log:test_07_delete_user.py:19 *************** 开始执行用例 ***************
INFO     log:http_client.py:40 接口请求地址 ==>> http://127.0.0.1/user/deleteUser
INFO     log:http_client.py:41 接口请求方式 ==>> POST
INFO     log:http_client.py:43 接口请求头 ==>> {
    "Content-Type": "application/json",
    "token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJyYmFjLXVzZXIiLCJpc3MiOiJsb25nY2hhbyIsImlhdCI6MTczMDc2ODM3NSwiZXhwIjoxNzMwNzcxOTc1LCJpZCI6MTg0NjAzMDEwOTI2MDAzNDA1MH0.f5KcR28S6oaSngKIua7ed6-vvGcSDhjRCqLHHZFU-o0"
}
INFO     log:http_client.py:45 接口请求 params 参数 ==>> {
    "userId": -1
}
INFO     log:index_user_api.py:15 接口实际返回的结果为==>>{
    "success": false,
    "code": 10007,
    "message": "当前用户不存在！",
    "time": "1730768379948",
    "path": null,
    "timestamp": null,
    "msg": null,
    "trackId": null
}
INFO     log:test_07_delete_user.py:29 *************** 恢复被删除用例状态 ***************
INFO     log:conftest.py:29 code ==>> 期望结果：10007， 实际结果：10007
INFO     log:conftest.py:30 返回信息 ==>> 期望结果：当前用户不存在！， 实际结果：当前用户不存在！
INFO     log:test_07_delete_user.py:33 *************** 结束执行用例 ***************